actions.md

```diff
@@ -143,7 +143,8 @@ These are the primary actions that a user of libaktualizr can perform through th
       - [x] Set new version to pending status after an OSTree update trigger (aktualizr_test.cc)
       - [x] Send EcuInstallationAppliedReport to server after an OSTree update trigger (aktualizr_test.cc)
       - [x] Uptane check for updates and manifest sends are disabled while an installation is pending reboot (aktualizr_test.cc)
-      - [ ] Trigger a system reboot if `reboot_after_install` is set in the configuration
+      - [ ] Trigger a system reboot at the end of the installation process for an ostree-based update mechanisms if `force_install_completion` is set in an uptane configuration
+      - [ ] Exit aktualizr and emulates a reboot in case of the fake package manager if `force_install_completion` is set in an uptane configuration
```

Comment:
Just a couple small suggestions here:
* "Trigger a system reboot at the end of an OSTree update if `force_install_completion` is set in the configuration"
* "Emulate a reboot for the fake package manager if `force_install_completion` is set in the configuration (aktualizr_test.cc)"

This is assuming that I understand correctly that the second item is covered by a test now. The same text should be in the comment for the test itself so there is an easy way to grep for it. (That's the best we have for now; hopefully we'll get to spend some time improving that mechanism sooner or later.)

Reply:
OK. I think, I got your idea and it makes sense.